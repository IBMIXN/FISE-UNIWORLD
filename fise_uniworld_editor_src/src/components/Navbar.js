import React from "react";
import { createStyles, makeStyles } from "@material-ui/core/styles";
import { Link as RouterLink } from "react-router-dom";
import AppBar from "@material-ui/core/AppBar";
import Toolbar from "@material-ui/core/Toolbar";
import Typography from "@material-ui/core/Typography";
import Button from "@material-ui/core/Button";

const useStyles = makeStyles((theme) =>
  createStyles({
    root: {
      flexGrow: 1,
    },
    title: {
      marginRight: theme.spacing(2),
    },
  })
);

const Navbar = () => {
  const classes = useStyles();
  return (
    <AppBar position="static" className={classes.root}>
      <Toolbar>
        <Typography variant="h6" className={classes.title}>
          UniWorld Editor
        </Typography>
        <Button color="inherit" component={RouterLink} to="/">
          Home
        </Button>
        <Button color="inherit" component={RouterLink} to="/eventrooms">
          Event Rooms
        </Button>
        <Button color="inherit" component={RouterLink} to="/lecturerooms">
          Lecture Rooms
        </Button>
      </Toolbar>
    </AppBar>
  );
};

export default Navbar;
