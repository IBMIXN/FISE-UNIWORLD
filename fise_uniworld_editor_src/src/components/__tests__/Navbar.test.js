import React from 'react';
import { shallow } from 'enzyme';
import Navbar from '../Navbar';

describe('Navbar test', () => {
  it('Navbar should match snapshot', () => {
    const component = shallow(<Navbar />);
    expect(component).toMatchSnapshot();
  });
});
