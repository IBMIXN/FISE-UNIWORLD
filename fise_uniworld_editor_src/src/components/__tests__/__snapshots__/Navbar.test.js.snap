// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar test Navbar should match snapshot 1`] = `
<WithStyles(ForwardRef(AppBar))
  className="makeStyles-root-1"
  position="static"
>
  <WithStyles(ForwardRef(Toolbar))>
    <WithStyles(ForwardRef(Typography))
      className="makeStyles-title-2"
      variant="h6"
    >
      UniWorld Editor
    </WithStyles(ForwardRef(Typography))>
    <WithStyles(ForwardRef(Button))
      color="inherit"
      component={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "propTypes": Object {
            "innerRef": [Function],
            "onClick": [Function],
            "replace": [Function],
            "target": [Function],
            "to": [Function],
          },
          "render": [Function],
        }
      }
      to="/"
    >
      Home
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      color="inherit"
      component={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "propTypes": Object {
            "innerRef": [Function],
            "onClick": [Function],
            "replace": [Function],
            "target": [Function],
            "to": [Function],
          },
          "render": [Function],
        }
      }
      to="/eventrooms"
    >
      Event Rooms
    </WithStyles(ForwardRef(Button))>
    <WithStyles(ForwardRef(Button))
      color="inherit"
      component={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "propTypes": Object {
            "innerRef": [Function],
            "onClick": [Function],
            "replace": [Function],
            "target": [Function],
            "to": [Function],
          },
          "render": [Function],
        }
      }
      to="/lecturerooms"
    >
      Lecture Rooms
    </WithStyles(ForwardRef(Button))>
  </WithStyles(ForwardRef(Toolbar))>
</WithStyles(ForwardRef(AppBar))>
`;
